### PR TITLE
[cmake] Only install VideoPlayer fontconfig configuration files on Apple

### PR DESCRIPTION
**Pull request by cjmayo**
## Motivation and Context
VideoPlayer fontconfig files are not needed, and removed downstream:
https://gitweb.gentoo.org/repo/gentoo.git/tree/media-tv/kodi/kodi-17.0_rc2.ebuild#n228
https://bugs.gentoo.org/show_bug.cgi?id=605712
## How Has This Been Tested?
Only tested on Linux

Don't know whether `if (APPLE)` is the wrong match.

## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -291,7 +291,9 @@ add_custom_target(pack-skins ALL
                   DEPENDS TexturePacker::TexturePacker export-files ${XBT_FILES})
 set_target_properties(pack-skins PROPERTIES FOLDER "Build Utilities")
 
-file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/system/players/VideoPlayer)
```
Comment (stefansaraev) [inline CMakeLists.txt]: this has to stay as is, system/players/VideoPlayer/ is used for other stuff like libdvdnav

**Comment by cjmayo** [comment]
OK that solves the question about 'if (APPLE)` then. Leave  system/players/VideoPlayer alone.

**Comment by stefansaraev** [comment]
project/cmake/installdata/ios/ is not the right path. in master cmake was moved to root, so cmake/installdata/ios/ it is. perhaps should also add to cmake/installdata/osx?

**Comment by cjmayo** [comment]
Sorry about that I forgot about the move. Obviously I should also have said this was tested on Krypton.
I've updated the paths but haven't yet tested on master.

**Comment by cjmayo** [comment]
Amend the commit title to reflect iOS only.

**Comment by stefansaraev** [comment]
thanks. this looks good to me but lets cc the people in charge @hudokkow @Memphiz fyi

**Comment by Memphiz** [comment]
Fine by me

**Comment by hudokkow** [comment]
jenkins build this please

**Comment by candrews** [comment]
Can this be backported to Krypton as well?

**Pull request merged**